### DISCUSSION TITLE
2.11.1

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#640] [Releases]
- Fix blurriness of main screen letter icons
- Fix icons sometimes disappearing after selection
- Fix status bar icons possibly being invisible on Android 5

<hr /><em>This discussion was created from the release <a href='https://github.com/TheLastProject/Catima/releases/tag/v2.11.1'>2.11.1</a>.</em>

**Comment by Altonss** [comment]
I just received this update on F-Droid and all 3 issues are fixed! Thank you for the great work!
